app/lib/zap/controller.js

```diff
@@ -419,7 +426,7 @@ class ZapController {
         this.neutrino.removeListener('exit', exitHandler)
         if (this.neutrino) {
           mainLog.warn('Graceful shutdown failed to complete within 10 seconds.')
-          this.neutrino.kill('SIGTERM')
+          this.neutrino.kill('SIGKILL')
```

Comment:
Consider extracting a constant since it's also used down below

Reply:
Where it's used below the usage is different.

Here, we are passing the `SIGKILL` signal. Where as below we are not passing a specific signal and using the default value of `SIGINT` which is defined in the `Neutrino.prototype.kill` method.

I'm not sure that there is any useful consolidation that can be done here given the usage of each call is deliberately different.

Reply:
Just realised that you are referring to the `SIGKILL` value, not the method call!

Reply:
Yes I'm referring to this:
[https://github.com/LN-Zap/zap-desktop/blob/92b21d9e09b0c1c90fce5d41fd0cb4af269c1293/app/lib/zap/controller.js#L498](https://github.com/LN-Zap/zap-desktop/blob/92b21d9e09b0c1c90fce5d41fd0cb4af269c1293/app/lib/zap/controller.js#L498)

Reply:
Although - I'm still not sure of the benefit of making this a constant in this instance...

Reply:
Maybe. I just usually extract if i have > 1 usages. And in general don't like string literals.

Reply:
I tend to agree if these are string that we are defining for our own purposes. But `SIGINT` and `SIGTERM` etc are standard process signals that can be passed to external processes. These are defined by node itself https://nodejs.org/api/process.html#process_signal_events